test/ft/wrap.cpp

```diff
@@ -0,0 +1,83 @@
+//
```

Comment:
Can we just add copyright/license stuff to this file, please, it's a requirement for BOOST license for all files to have it

Reply:
I added the license to my test, but the test `actions_process_n_defer.cpp` doesn't have it, so I think you should check it out.